extensions/cornerstone/src/id.js

```diff
@@ -1,3 +1,6 @@
-const id = 'org.ohif.cornerstone';
+import packageJson from '../package.json';
```

Comment:
Will this cause us any issues?

The ugly thing here is that we are including the package json of all extensions/modes in source now. But I get that you don't want to have the ID defined twice.